doc/whats_new.rst

```diff
@@ -0,0 +1,2376 @@
+0.8.2.dev
```

Comment:
there is a large diff in the whats_new, did something go wrong with the rebase?

Reply:
Hmmm, yes the whats_new shouldn't even be there. The file was deleted by #3049 and separated in different files under `doc/changes`.